app/uk/gov/hmrc/servicedependencies/connector/TeamsAndRepositoriesConnector.scala

```diff
@@ -54,8 +54,8 @@ class TeamsAndRepositoriesConnector @Inject()(
         .map(TeamsForServices.apply)
     }
 
-  def getAllRepositories(implicit hc: HeaderCarrier): Future[Seq[RepositoryInfo]] =
-    httpClient.GET[Seq[RepositoryInfo]](s"$teamsAndRepositoriesApiBase/api/repositories")
+  def getAllRepositories(archived: Option[Boolean])(implicit hc: HeaderCarrier): Future[Seq[RepositoryInfo]] =
```

Comment:
would it be better to just have a straight boolean with default rather than option[boolean]?

Reply:
there are 3 options here - None, being both archived and non-archived (i.e applying no filter).